src/Domain/CoordinateMaps/TimeDependent/Translation.hpp

```diff
@@ -11,28 +11,93 @@
 class FunctionOfTime;
-}  // namespace FunctionsOfTime
-}  // namespace domain
+}  // namespace domain::FunctionsOfTime
 namespace PUP {
 class er;
 }  // namespace PUP
 /// \endcond
 
-namespace domain {
-namespace CoordinateMaps {
-namespace TimeDependent {
+namespace domain::CoordinateMaps::TimeDependent {
 /*!
  * \ingroup CoordMapsTimeDependentGroup
- * \brief Translation map defined by \f$\vec{x} = \vec{\xi}+\vec{T}(t)\f$.
+ * \brief Translation map defined by \f$\vec{x} = \vec{\xi}+F(r)\vec{T}(t)\f$.
  *
- * The map adds a translation, \f$\vec{T}(t)\f$, to the coordinates
- * \f$\vec{\xi}\f$, where \f$\vec{T}(t)\f$ is a FunctionOfTime.
+ * \details The map adds a translation, \f$F(r)\vec{T}(t)\f$, to the coordinates
+ * \f$\vec{\xi}\f$, where \f$\vec{T}(t)\f$ is a FunctionOfTime and\f$F(r)\f$ is
+ * a 1D radial MathFunction. The radius of each point is found by subtracting
+ * the center map argument from the coordinates \f$\vec{\xi}\f$ or the target
+ * coordinates \f$\vec{\bar{\xi}}\f$. The Translation Map class is overloaded so
+ * that the user can choose if a radial depedence is needed for their problem.
+ * If a radial dependence is not specified, this sets \f$F(r) = 1\f$.
+ *
+ * ### Mapped Coordinates
+ *
+ * The translation map translates the coordinates \f$\vec{\xi}\f$ to the
+ * target coordinates
+ * \f{equation}{
+ * \vec{\bar{\xi}} = \vec{\xi} + F(r)\vec{T}(t)
+ * \f}
+ *
+ * If you only supply a FunctionOfTime to the constructor of this class, the
+ * radial function will be set to 1.0 causing a uniform translation for your
+ * coordinates. If a FunctionOfTime, MathFunction, and map center are passed in,
+ * the radius will be found through
+ * \f{equation}{
+ * r = |\vec{\xi} - \vec{c}|
+ * \f}
+ * where r is the radius and \f$\vec{c}\f$ is the center argument.
+ *
+ * ### Inverse Translation
+ * The inverse translation translates the coordinate \f$\vec{\bar{\xi}}\f$ to
+ * the original coordinates
+ * \f{equation}{
+ * \vec{\xi} = \vec{\bar{\xi}} - F(root)\vec{T}(t)
+ * \f}
+ * However, we need to do a root find to see where F(r) and \f$\vec{T}(t)\f$
+ * cross zero. This is done by bracketing and finding the root of
+ * \f{equation}{
+ * \bar{r}^2 + F(x) (A + |\vec{T}(t)| F(x)) - x^2
+ * \f}
+ * This equation comes from the SpEC implementation where x is the root guess
+ * and A is the Root Factor given by
+ * \f{equation}{
+ * A = -2.0 (\vec{\xi} - \vec{c}) \cdot \vec{T}(t)
+ * \f}
+ *
+ * ### Frame Velocity
+ *
+ * The frame velocity is given by
+ * \f{equation}{
+ * \vec{v} = \frac{\vec{dT}(t)}{dt} * F(r)
+ * \f}
+ * where \f$\frac{\vec{dT}(t)}{dt}\f$ is the first derivative of the
+ * FunctionOfTime.
+ *
+ * ### Jacobian
+ * The jacobian is computed through either first or second derivatives of the
+ * radial MathFunction based on your radius. When the radius is bigger
+ * than 1.e-13, the jacobian is given by:
+ * \f{equation}{
+ * J^{i}_{j} = \frac{dF(r)}{dr} * \vec{T}(t) * \frac{(\vec{\xi} - \vec{c})}{r}
```

Comment:
$\LaTeX$ is weird so for the `J^{i}_{j}` ( $J^{i}\_{j}$ ), it actually renders the `i` and `j` above each other, rather than how we normally write indices. To achieve that you have to do `{J^{i}}_{j}` ( ${J^{i}}_{j}$ ).

 Also no `*`. This notation is a bit confusing because you're mixing index notation with vector notation. I think it'd be clearer if you wrote this as 

${J^{i}}_{j} = \frac{dF(r)}{dr} T(t)^i \frac{(\xi_j - c_j)}{r}$

Same (on all changes) for the other jacobian below.